doc/nrf/installation/install_ncs.rst

```diff
@@ -611,16 +618,15 @@ To install the |NCS| system-wide, complete the following steps:
 Installation with Toolchain Manager
 ***********************************
 
-.. note::
+Toolchain Manager is a tool available from `nRF Connect for Desktop`_, a cross-platform tool that provides different applications that simplify installing the |NCS|.
```

Comment:
```suggestion
Toolchain Manager is a tool available from `nRF Connect for Desktop`_.
It is a cross-platform tool that provides different applications simplifying the installation of the |NCS|.
```

Reply:
This edit suggests the cross-platform tool is the Toolchain Manager, while that sentence refers to nRF Connect for Desktop. Rephrased.

Reply:
You are so right.